### ISSUE TITLE
Possible to support origin with no path?

### ISSUE THREAD

**Issue by sheddup** [Reeywhaar/corser#2]
Hey, great extension it works well but I've run into an issue, I have requests originating from http://website.com 
The origin has no path value so I can't add it to the rule set because its not a valid match pattern, I tried using http://website.com/ as a rule but it does not work.  
![pic](https://user-images.githubusercontent.com/5480335/33530833-5093b060-d86c-11e7-9815-434b55e56424.png)

**Comment by reeywhaar** [comment]
Hi, @sheddup 
I have exact this usecase, rules must be working.
The thing is, rules must end with slash, so they conform to [Mozilla match patterns](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Match_patterns)

So i have for example:
```
http://website.com/
http://website.com/*
-
*://*.google.com/*
```

`http://website.com` is invalid pattern, I'm afraid.

Hope it will help!

**Comment by sheddup** [comment]
No, ending with a slash doesn't work as stated in OP, the exact origin doesn't end with a slash so it would not match.
